<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NumKeyBoard</class>
 <widget class="QWidget" name="NumKeyBoard">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>160</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>240</width>
    <height>160</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>240</width>
    <height>160</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>248</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="pushButton_7">
      <property name="minimumSize">
       <size>
        <width>78</width>
        <height>26</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>78</width>
        <height>26</height>
       </size>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_8">
      <property name="minimumSize">
       <size>
        <width>78</width>
        <height>26</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>78</width>
        <height>26</height>
       </size>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_9">
      <property name="minimumSize">
       <size>
        <width>78</width>
        <height>26</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>78</width>
        <height>26</height>
       </size>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>248</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="minimumSize">
       <size>
        <width>78</width>
        <height>26</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>78</width>
        <height>26</height>
       </size>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_5">
      <property name="minimumSize">
       <size>
        <width>78</width>
        <height>26</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>78</width>
        <height>26</height>
       </size>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_6">
      <property name="minimumSize">
       <size>
        <width>78</width>
        <height>26</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>78</width>
        <height>26</height>
       </size>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>248</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="pushButton_1">
      <property name="minimumSize">
       <size>
        <width>78</width>
        <height>26</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>78</width>
        <height>26</height>
       </size>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="minimumSize">
       <size>
        <width>78</width>
        <height>26</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>78</width>
        <height>26</height>
       </size>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="minimumSize">
       <size>
        <width>78</width>
        <height>26</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>78</width>
        <height>26</height>
       </size>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>248</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="pushButton_dou">
      <property name="minimumSize">
       <size>
        <width>78</width>
        <height>26</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>78</width>
        <height>26</height>
       </size>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_0">
      <property name="minimumSize">
       <size>
        <width>78</width>
        <height>26</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>78</width>
        <height>26</height>
       </size>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_true">
      <property name="minimumSize">
       <size>
        <width>78</width>
        <height>26</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>78</width>
        <height>26</height>
       </size>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>248</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="pushButton_detele">
      <property name="minimumSize">
       <size>
        <width>78</width>
        <height>26</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>78</width>
        <height>26</height>
       </size>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_clear">
      <property name="minimumSize">
       <size>
        <width>78</width>
        <height>26</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>78</width>
        <height>26</height>
       </size>
      </property>
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_19">
      <property name="minimumSize">
       <size>
        <width>78</width>
        <height>26</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>78</width>
        <height>26</height>
       </size>
      </property>
      <property name="text">
       <string>C</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pushButton_1</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>pushButton_5</tabstop>
  <tabstop>pushButton_6</tabstop>
  <tabstop>pushButton_7</tabstop>
  <tabstop>pushButton_8</tabstop>
  <tabstop>pushButton_9</tabstop>
  <tabstop>pushButton_dou</tabstop>
  <tabstop>pushButton_0</tabstop>
  <tabstop>pushButton_true</tabstop>
  <tabstop>pushButton_detele</tabstop>
  <tabstop>pushButton_clear</tabstop>
  <tabstop>pushButton_19</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
